andela/ah-frontend-valkyrie
# #162633467 Implement article pagination

**What does this PR do?**

Implements pagination functionality on articles

**Description of Task to be completed?**

- Create a pagination component.
- Create methods for paginating articles.
- Modify article actions to cater for pagination requirements

**How should this be manually tested?**

- Run `npm start` to run the application.
- Click on any item on the pagination bar to ensure that you can navigate to a different page 

**What are the relevant pivotal tracker stories?**

[#162633467](https://www.pivotaltracker.com/story/show/162633467)
